atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/api/watch/LockWatchEventCacheIntegrationTest.java

```diff
@@ -296,30 +296,72 @@ public void cacheClearedOnSnapshotUpdate() {
     }
 
     @Test
-    public void getEventsForTransactionsReturnsSnapshotWithOldEvents() {
-        createEventCache(3);
+    public void getEventsForTransactionsNoCondensing() {
         setupInitialState();
-        eventCache.processGetCommitTimestampsUpdate(COMMIT_UPDATE, SUCCESS);
-        eventCache.removeTransactionStateFromCache(START_TS);
-        verifyStage();
-
+        eventCache.processStartTransactionsUpdate(ImmutableSet.of(), SUCCESS);
         eventCache.processStartTransactionsUpdate(TIMESTAMPS_2, SUCCESS_2);
         verifyStage();
 
-        TransactionsLockWatchUpdate results = eventCache.getUpdateForTransactions(TIMESTAMPS_2, Optional.empty());
+        // Client is behind and needs a snapshot, but no compression can be done as the first transaction is at the
+        // first version
+        TransactionsLockWatchUpdate results =
+                eventCache.getUpdateForTransactions(Sets.union(TIMESTAMPS, TIMESTAMPS_2), Optional.empty());
```

Comment:
Can we make these tests more legible by not using seemingly magic numbers?
I.e.,
```suggestion
               eventCache.getUpdateForTransactions(ImmutableSet.of(START_TS, TS_16), Optional.empty());
```
and then using those in the asserts, and similarly below not using the magic `LockWatchVersion` but call `logId` and `lastKnownVersion` on the relevant updates?

Reply:
Done